### PR TITLE
Simplify hello world and use wallet provider

### PR DESCRIPTION
**Pull request by AurelienFT**

### CODE REVIEW

File: helloworld/front/react/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+        registerAndSetProvider();
+    }, []);
+
+    const callHelloWorld = async () => {
+        try {
+            if (!nodeUrls || !account) {
+                return;
+            }
+            let client = await ClientFactory.createClientFromWalletProvider(nodeUrls, account);
```
Comment (gregLibert) [inline helloworld/front/react/src/App.tsx:41]: Why do you need to pass something to the walletProvider ? You should be able to directly call fromWalletProvider without arguments and let this function call wallet provider to extract the needed info.
And yes, I renamed the function as ClientFactory.fromWalletProvider is self explanatory I guess :-)

**Review by gregLibert — COMMENTED** [review]
TO be also discussed in live meeting.

File: helloworld/front/react/src/App.tsx (hunk 1 above)
Comment (AurelienFT) [reply helloworld/front/react/src/App.tsx:41]: Done :)

Comment (gregLibert) [reply helloworld/front/react/src/App.tsx:41]: fixed on massaweb3:  nodeUrls replaced by provider. The rest of the simplificatino is to be done at a component level.

**Review by Ben-Rey — APPROVED** [review]

**Pull request merged**